modules/ROOT/pages/multi-factor-authentication.adoc

```diff
@@ -0,0 +1,142 @@
+
+Users who use single sign-on (SSO) cannot configure multi-factor authentication for their accounts. Instead, SSO users should configure multi-factor authentication through their identity provider.
+
+When multi-factor authentication is enabled, users must verify their Anypoint Platform credentials using at least one of the following authentication methods:
+
+* Third-party TOTP authenticator apps
+* Built-in authenticator
+* Security key
+* Salesforce Authenticator
+
+
+== Multi-factor Authentication Configurations
+
+You can configure at least one of the following verification methods to comply with multi-factor authentication requirements:
+
+Third-party TOTP authenticator apps:: Registers an authenticator app to create verification codes that you provide when logging in to Anypoint Platform.
+
+Built-in authenticator:: Registers a physical authentication device, such as Touch ID or Windows Hello, to verify your identity when logging in to Anypoint Platform.
+
+Security key:: Registers a USB security key, such as Yubico YubiKey or Google Titan Security Key, to your account. The device is then authorized to create verification codes that you provide when logging in to Anypoint Platform.
+
+Salesforce Authenticator:: Registers the Salesforce Authenticator mobile app to create verification codes that you provide when logging in to Anypoint Platform.
+
+== Enable Multi-factor Authentication for All User Accounts
+
+For enhanced security, you can require all users in your organization to use multi-factor authentication. Users must configure their authentication method when they log in to Anypoint Platform after you initially enable multi-factor authentication.
+
+[NOTE]
+--
+To prevent service disruptions, identify any accounts that are used to programmatically call Anypoint Platform, or _service accounts_. You must either configure these calls using the xref:connected-apps-overview.adoc[Connected Apps] feature or add these accounts to the *Exempt Accounts* section when configuring multi-factor authentication requirements. 
+--
+
+. Log in to Anypoint Platform using an account that has the Organization Administrator permission.
+. Navigate to *Access Management*.
+. In the navigation menu, click *Multi-factor auth*.
+. Select *Required*.
+. If you want to exclude users, such as service accounts, from multi-factor authentication, add users to the *Exempt Accounts* section.
+. Click *Save*. +
+All users in your organization are now required to configure a multi-factor authentication method when they log into Anypoint Platform. +
+If a user already has multi-factor authentication configured for their account, they are not prompted to configure another method.
+
+=== Exempt Users from Multi-factor Authentication
+
+Accounts that make programmatic calls to Anypoint Platform, or service users, should be exempt from multi-factor authentication since they cannot provide verification aside from a username and password.
```

Comment:
```suggestion
Service users or accounts that make programmatic calls to Anypoint Platform should be exempt from multi-factor authentication because they cannot provide verification aside from a username and password.
```